products/amazon-corretto.md

```diff
@@ -44,8 +44,8 @@ releases:
     # Not yet announced on https://aws.amazon.com/corretto/faqs/,
     # using the next 1th or the month from Java 21 release (that what they usually do).
     eol: 2023-10-01
-    latest: "20.0.0.36.1"
-    latestReleaseDate: 2023-02-21
+    latest: "20.0.1.9.1"
```

Comment:
We may consider switching to the github_release auto method to have more accurate dates.

Reply:
#2867